spec/collections/djornl/djornl_edge.yaml

```diff
@@ -13,7 +13,7 @@ schema:
   title: Arabidopsis gene-gene or gene-phenotype edge
   description: Generic gene-to-gene or gene-to-phenotype edge for Dan Jacobson Arabidopsis data
   type: object
-  required: [score, edge_type, _from, _to, _key]
+  required: [score, edge_type, _from, _to, _key, directed]
```

Comment:
edges must now have the `directed` field, which is either true or false (default: false)